### PR TITLE
New Log Guide: keep control on sensitive data flowing in logs

### PR DESCRIPTION
**Pull request by pcarioufr**
### Preview

- https://docs-staging.datadoghq.com/pcarioufr/restrict_access_sensitive_data/logs/guide/
- https://docs-staging.datadoghq.com/pcarioufr/restrict_access_sensitive_data/logs/guide/restrict-access-to-sensitive-data-with-rbac/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/guide/logs-restrict-sensitive.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    text: "Fix sensitive data leaking with Advanced Log Collection"
+  - /agent/guide/autodiscovery-management/
+    tag: Documentation
+    text: "Exclude containers from Log Collection with Autodiscovery"
+
+---
+
+## Introduction
+
+Logs are prone to hold sensitive data, and therefore need a specific care from a compliance standpoint. There are two approaches regarding sensitive data management:
```
Comment (NoisomePossum) [inline content/en/logs/guide/logs-restrict-sensitive.md:19]: need specific care

File: content/en/logs/guide/logs-restrict-sensitive.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+If you have to patch your archives to remove sensitive data, refer to the documented [format of archives][33] generated by Datadog.
+
+
+### Reach out to support 
+
+In case you have specific compliance questions or need a helping hand in fixing this, feel free to reach out to our support (support@datadoghq.com). Helpful information to support you are:
+
+* The sensitive outline query. Or anything that could define sensitive data such as a timerange, a service or an environment,
+* Whether the sensitive data still leaking into Datadog,
```
Comment (NoisomePossum) [inline content/en/logs/guide/logs-restrict-sensitive.md:106]: Whether the sensitive data **is** still

**Review by apigirl — COMMENTED** [review]
pushed grammatical feedback when I fixed the md issue

**Comment by pcarioufr** [comment]
@apigirl Since your last review, I brought a bit of structure on the Guide Page, following APM's Guides page model.

**Comment by pcarioufr** [comment]
@apigirl I also changed the title, along with the name of the file, for something less focused on the Query Based Access Controls part (which is only one optional step in the bigger picture).

**Comment by apigirl** [comment]
> @apigirl I also changed the title, along with the name of the file, for something less focused on the Query Based Access Controls part (which is only one optional step in the bigger picture).

I actually changed it again - let me know what you think

**Review by apigirl — APPROVED** [review]

**Pull request merged**